src/content/docs/style-guide/writing-strategies/contribute-to-site.mdx

```diff
@@ -0,0 +1,376 @@
+      alt="Click the pencil button to go into edit mode"
+      src={styleguideCommitChanges}
+  />
+
+Once you're ready to save your progress, click the **Commit changes...** button. This brings you to a page that looks like this:
+
+  <img
+    title="Write a commit message"
+    alt="Click the pencil button to go into edit mode"
+    src={styleguideCommitMessage}
+  />
+
+Let's break down what this form is asking:
+
+* **Commit message**: Required. A commit message is a brief explanation of what you changed. For example, one might read like: `Updated requirements and added content to install procedures`
+* **Extended description**: Optional, but helps the Docs team review your changes faster. This space  can add additional context about the change: `Existing install procedures caused customer confusion`.
+* Always choose **Create a new branch for this commit...**. We won't accept any contribution made directly to `develop`.
+
+There are no hard and fast rules about how often you should commit. You can make as many or as few commits as you'd like.
+
+    </Step>
+    <Step>
+        ### Navigate your new pull request
+
+Your changes—that is, your  branch and its commits—will be packaged into a pull request. This essentially creates a working space for you and anyone who wants to review your work. Here's an example pull request page:
+
+  <img
+    title="Your PR and you"
+    alt="Your pull request is a working space for your contributions"
+    src={styleguidePRpage}
+  />
+
+The two main areas are **Conversation** and **Files changed**.
+
+* The **Conversation** tab is where you can see comments, like feedback from the Docs team. 
+    * You can manage the status of your pull request as well. Scroll to **Projects** in the right nav and choose the status of your pull request. If it's a work in progress, put in **Drafts**. When you're finished, put it in **Hero to triage**.
+      <img
+      title="Change the status of your project"
+      alt="Your pull request is a working space for your contributions"
+      src={styleguideManageProjects}
+      />
+    * Alternatively, if you scroll to the bottom of the pull request page, you can find the comment box. Many contributors leave updates about the status of their work, like when they'd like their contributions reviewed and released.
+* The **Files changed** tab lets you review or continue your work. Click the ellipses to open up the text edit view. You can continue to make commits from this tab. 
+      <img
+        title="Your PR workspace"
+        alt="The files changed tab lets you continue work on a doc"
+        src={styleguideContinueWorkPR}
+      />
+    </Step>
+    <Step>
+    ### Wait for a docs review
+
+  A tech writer will assign the pull request to themself when it's in review. You can track the status of your pull request from **Projects** in the right nav. Make sure to return to your pull request for any changes requested or follow-up questions from a writer. 
```

Comment:
Up to this point, you've used "docs team". Do we need to draw a connection here between "tech writer" and "docs team"